chapters/annotations.tex

```diff
@@ -383,20 +383,25 @@ \section{Annotations for Simulations}\label{annotations-for-simulations}
 \end{center}
 
 \begin{annotationdefinition}[experiment]
-% henrikt-ma 2021-04: Seems strange to allow 'experiment' completely without list of options -- what would it mean?
 \begin{synopsis}[grammar]\begin{lstlisting}
```

Comment:
Why not go all in for the record style of presentation, simply skipping the grammar?  GitHub doesn't allow me to make a complete suggestion, but the idea is to simply language in the `synopsis`, and replace the piece of grammar by the definition of the record.
```suggestion
\begin{synopsis}\begin{lstlisting}
```

Reply:
This change has been applied